src/test/java/org/broadinstitute/hellbender/tools/exome/allelefraction/AFSamplersTest.java

```diff
@@ -0,0 +1,119 @@
+package org.broadinstitute.hellbender.tools.exome.allelefraction;
+
+import org.apache.commons.math3.random.RandomGenerator;
+import org.apache.commons.math3.random.RandomGeneratorFactory;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.stream.Collectors;
+
+import static org.testng.Assert.*;
+
+/**
+ * Created by davidben on 11/25/15.
+ */
+public class AFSamplersTest {
+    private static final int RANDOM_SEED = 42;
```

Comment:
need to fix up the modifiers for the fields in this class. check order of modifiers (private static final) and change field names to BLOCK_UPPERCASE as appropriate.